client/src/Component/SMSModal.js

```diff
@@ -1,23 +1,54 @@
-/* eslint-disable jsx-a11y/label-has-for */
+
 import { useState } from "react";
 import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/Button";
 
+
+
+
 function SMSModal(props) {
     const [show, setShow] = useState(false);
     const hideModal = () => setShow(false);
     const showModal = () => setShow(true);
 
+    let localMessage = localStorage.getItem("message");
+    if (localMessage === null) {
+      localMessage = "Hi, this is Bikes For Refugees. Bikes are now available for you to pick up. Please click LINK to choose a pick up date. For any questions, please contact 07900000000.";
+    }
+    const [message, setMessage] = useState(localMessage);
+
+    const handleChange = (e) => {
+      localStorage.setItem("message", e.target.value);
+      setMessage(e.target.value);
+    };
+
 
     function handleSubmit () {
-      console.log(props.checkedCheckboxes);
-      hideModal();
-      }
+
+      const newMessageRequest = {
+          ids : props.selectedClients,
+          message: message,
```

Comment:
```suggestion
          message,
```
Nitpick: We could shorten this using the [object property shorthand](https://alligator.io/js/object-property-shorthand-es6/)

Reply:
🤯